OneMore/Commands/File/ImportCommand.cs

```diff
@@ -509,12 +509,6 @@ private async Task ImportPdfFile(string filepath, bool append, CancellationToken
 
 		private async Task ImportMarkdown(string filepath)
 		{
-			if (filepath.IndexOf(' ') >= 0)
```

Comment:
There's a reason why spaces were not allowed. If there are spaces in the path, the wikilinks extension would break. There may be another way to solve that but this certainly breaks it.

Reply:
Ping. This still needs to be resolved a different way